backend/routes/products.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const router = express.Router();
+const productQueries = require('../queries/products');
+const {handleErrors} = require('./helpers/helpers');
+
+/* GET all product by Brand Id */
+router.get('/:brand_id/all', async (req, res, next) => {
+
+    let brand_id = req.params.brand_id
+
+    try {
+        let productByBrand = await productQueries.getProductsByBrand(brand_id)
+        res.status(200).json({
+            message: `Success retrieved all products for brand with id #${brand_id}`,
+            payload: productByBrand
+        })
+
+    } catch (err) {
+        handleErrors(res, err)
+    }
+})
+
+/* GET all product by type Id */
+router.get('/:type_id', async (req, res) =>{
```

Comment:
Probably better to have this route like :
router.get('/type/:type_id', async (req, res) =>{
otherwise the two fellowing routers will hit this one instead